src/main/java/carRacing/Main.java

```diff
@@ -3,14 +3,14 @@
 public class Main {
 
     public static void main(String[] args) {
-        int carCount = InputView.initCarCount();
-        RacingGame racingGame = new RacingGame();
-        racingGame.initRacingCars(carCount);
+        String nameStr = InputView.initCarInfo();
```

Comment:
```suggestion
        String names = InputView.initCarInfo();
```
변수 앞에 타입이 있으니 굳이 변수명에는 타입이 없어도 되지 않을까요?
추후 타입이 변경된다면